### DISCUSSION TITLE
Client-Side Bug with Initialization in Multiplayer

### DISCUSSION THREAD

**Discussion by 4-S-H** [FakeFishGames/Barotrauma#15436] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

1. When the client enters a server, there is a chance that no character is spawned, all crew are shown as observers, and the crew list on the left is empty. However, the server log and crew radio communication are synced as normal. The game is basically stuck in this state.

2. The second attempt to join would instantly crash the game upon clicking the "enter" button with no bug report produced. (Happened consistently to me if 1 happens)

(the order is "ignore")
![image](https://github.com/user-attachments/assets/6467148c-3275-48f8-b7cc-971a616cd33c)
![image](https://github.com/user-attachments/assets/ec944246-567f-4d77-a14f-f1ba3cfdeb22)
![image](https://github.com/user-attachments/assets/83942ce7-e0ad-466c-b534-d5f1b44371b4)
![image](https://github.com/user-attachments/assets/45b9cd90-6c22-47c9-9c9e-74d635e60c28)

### Reproduction steps

1. Enter a server (maybe one with a lot of events going on? It happens more often to me mid-round than at the beginning of the round)
2. There is a chance the bug will happen

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
![20241231135006](https://github.com/user-attachments/assets/efa1bf94-c0d2-48c6-83e4-d208c3be62aa)

**Comment by 4-S-H** [comment]
occurs again. last console messagses:
![image](https://github.com/user-attachments/assets/aeab67e4-4e79-4272-94dd-eb9d5bf39040)
